tools/run-task

```diff
@@ -132,25 +113,22 @@ function listrify(steps, {concurrent = false, verbose = false} = {}) {
             if (Array.isArray(task)) {
                 return { title, task: () => listrify(task.map(_task => {
```

Comment:
I'm not sure if it makes a huge difference, or any difference, but there are a few references to `title` in this function when elsewhere the property name has changed to `description`. Might be slightly confusing